bootstrap/_rpm_common.sh

```diff
@@ -20,11 +20,15 @@ $tool install -y \
   git-core \
   python \
   python-devel \
-  python-virtualenv \
-  python-devel \
+  python-setuptools \
   gcc \
   dialog \
   augeas-libs \
   openssl-devel \
   libffi-devel \
   ca-certificates \
+
+if ! $tool install -y python-virtualenv ; then
+    easy_install pip
+    pip install virtualenv
```

Comment:
@devnsec-com yeah i believe you will be re-adding python 2.6 at GA release time.. so you probably need to add the note that EPEL repo is required for centos/rhel particular 6.x releases if you want python-virtualenv package for python 2.6